internal/cli/atlas/accesslogs/list.go

```diff
@@ -32,8 +32,8 @@ import (
 const (
 	success      = "success"
 	fail         = "fail"
-	listTemplate = `HOSTNAME	CLUSTER NAME	AUTH RESULT	LOG LINE {{range .AccessLogs}}
-{{if .Hostname}}{{.Hostname}} {{else}}N/A{{end}}{{.Hostname}}	{{if .ClusterName}}{{.ClusterName}} {{else}}N/A{{end}}	{{.AuthResult}}	{{.LogLine}}{{end}}
+	listTemplate = `HOSTNAME	AUTH RESULT	LOG LINE {{range .AccessLogs}}
```

Comment:
have we removed it?

Reply:
This is answered [here](https://github.com/mongodb/mongodb-atlas-cli/pull/1798#pullrequestreview-1354998482) (previous PR comment)